umbraco/Umbraco-CMS
# merge user/current/logins and user/current/login-providers

### Prerequisites


### Description
There are currently 2 endpoints that return information about login providers.
- the actual linked logins at /user/current/logins
- and the list of possible providers with status at /user/current/login-providers

Since our DB model only supports 1 login per provider per user, we can merge these endpoints reducing a call by the frontend.

Also deleted an unused request model.

### Breaking information
Removing the /user/current/logins is technically breaking between the RC's but since this endpoint is most likely not used by implementors, it is considered safe to remove.

### Testing
- Configure one more more external login providers
- Check they all still show up in the login-providers endpoint
- Login or link with a provider
- Check whether the login-providers endpoint shows the update information including the providerkey
